functions-and-scope.js

```diff
@@ -41,13 +67,28 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // Log het antwoord in de terminal.
 
 // ---- Verwachte uitkomst: 6.642857142857143
+let averageGrade = 0;
+function calculateAverageGrade(grades){
+averageGrade = 0;
+    for (let i = 0; i<grades.length; i++){
+        averageGrade = averageGrade + grades[i];
```

Comment:
Dit kun je nog iets korter schrijven
```suggestion
let averageGrade = 0;
function calculateAverageGrade(grades){
averageGrade = 0;
    for (let i = 0; i<grades.length; i++){
        averageGrade += grades[i];
```